spec/controllers/articles_controller_spec.rb

```diff
@@ -1,47 +1,41 @@
 require 'spec_helper'
 
 RSpec.describe ArticlesController, type: :controller do
-  describe '#index' do
-    describe 'show the list to articles saved' do
+  describe 'GET #index' do
+    it 'lists all the articles' do
     end
   end
 
-  describe '#show' do
-    it 'show the articles' do
+  describe 'GET #show' do
+    it 'show the article' do
     end
   end
 
-  describe '#new'do
-    it 'create new post' do
+  describe 'GET #new'do
+    it 'shot the  new post' do
     end
   end
 
-    describe ' #create' do
-        context 'when invalid an article' do
-          it 'return article false ' do
-          end
-        end
-        context 'when valid an article' do
-          it 'return for create new article' do
-          end
-        end
+  describe 'POST #create' do
+    it 'article is valid' do
+      expect(subject).to redirect_to(@article)
     end
-
-    describe '#update' do
-      describe 'update a record that already exists,'do
-        context 'when it finds Article' do
-          it 'redirect to article' do
-          end
-        end
-        context 'when not it find article' do
-          it 'redirect to new article page' do
-          end
-        end
-      end
+    it 'article is invalid' do
+      expect(subject).to render_template(:new)
     end
+  end
 
-    describe '#destroy' do
-      it 'destroy the articles' do
+  describe 'PUT #update' do
+    it 'finds the requested article' do
```

Comment:
Same as above. Here the spec description is not telling what's going on with the expectation.